src/bootstrap/src/utils/tests/mod.rs

```diff
@@ -98,6 +98,14 @@ impl ConfigBuilder {
         self
     }
 
+    pub fn config_toml(mut self, config_toml: &str) -> Self {
+        let toml_path = self.directory.join("bootstrap.toml");
+        std::fs::write(&toml_path, config_toml).unwrap();
+        self.args.push("--config".to_string());
```

Comment:
This shouldn't be needed, right? The default file should be loaded automatically.

Reply:
Hmm, but the default file shouldn't be loaded from `src`? https://github.com/rust-lang/rust/blob/b00998aaa50a78b1e45fb107a6c0cfd3f1dc44dd/src/bootstrap/src/core/config/config.rs#L1841 it should be primarily loaded from the current working directory.